test/e2e-go/restAPI/restClient_test.go

```diff
@@ -1965,6 +1966,11 @@ int 1`
 	a.Equal(expectedResult, resp)
 }
 
+func constToPtr[T any](constVar T) *T {
+	var localVar = constVar
+	return &localVar
```

Comment:
nit: can you return `&constVar` instead?

Reply:
right, modified in [use stackvalue, deprecate tealvalue, modify testcase](https://github.com/algorand/go-algorand/pull/5440/commits/4a2765ef85d1ebc669bdc52d3229f17ff10387a8)